workers/unity/Packages/com.improbable.gdk.core/Utility/EntityTemplate.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Improbable.Worker.Core;
+
+namespace Improbable.Gdk.Core
+{
+    public class EntityTemplate
```

Comment:
This isn't a template though. This is a single use wrapper. Template makes it sound like it can be reused.

Reply:
Discussed offline. Won't fix for now.